modules/update-upgrading-cli.adoc

```diff
@@ -78,6 +56,18 @@ VERSION IMAGE
 4.10.0  quay.io/openshift-release-dev/ocp-release@sha256:9c5f0df8b192a0d7b46cd5f6a4da2289c155fd5302dec7954f8f06c878160b8b
```

Comment:
we should update this output to match what a current 4.10 `oc` would show against a 4.10 cluster.  This will include your current channel and any available alternatives.  Then the following `channel` subcommand call can be "If you need to change the channel based on your organization requirements..." or some such (no need to call the `channel` subcommand if you're already in your desired channel).